bin/foreman-tasks

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+foreman_root = '/usr/share/foreman'
+require File.expand_path('./config/application', foreman_root)
+Dynflow::Rails::Daemon.new.run_background(parser.command, parser.options)
```

Comment:
`undefined local variable or method 'parser' for main:Object (NameError)`

Reply:
So much for pushing in hurry… updated